install.py

```diff
@@ -957,8 +957,50 @@ def sync_users():
             system(cmd)
 
 
+def git_check():
+    if not sys.stdout.isatty():
+        return True
+    try:
+        git_process = subprocess.Popen(
```

Comment:
You don't need to communicate with stdin, so just use `subprocess.check_output` and remove all that `PIPE` crap.